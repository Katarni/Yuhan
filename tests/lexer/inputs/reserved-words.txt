a = false
a and f
int for
forrr
