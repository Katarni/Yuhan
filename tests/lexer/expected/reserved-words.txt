identifier a 0:0
lvalue binary operator = 0:2
numeric literal 0 0:4
identifier a 1:0
rvalue binary operator and 1:2
identifier f 1:6
reserved word int 2:0
reserved word for 2:4
identifier forrr 3:0
